Components/Input/input.css

```diff
@@ -0,0 +1,65 @@
+:root {
+    --success: #008000;
+    --warning: #FFA500;
+    --error: #FF0000;
+    --medium-font-size: 1.5rem;
+}
+
+.input-section {
+    margin-top: auto;
+    margin-bottom: auto;
+}
+.input{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.input-box {
+    padding: 1rem;
+    font-size: var(--medium-font-size);
+    border: 2px solid var(--primary-color);
+   border-radius: 15px;
+}
+
+.basic {
+    border: 2px solid var(--primary-color);
+}
+
+.success {
+    border: 2px solid var(--success);
+
+}
+
+.warning {
+    border: 2px solid var(--warning);
+
+}
+
+.error {
+    border: 2px solid var(--error);
+
+}
+.input-label-success{
```

Comment:
Better to use Prettier for Code Formatting

Reply:
Done that 👍🏻